### DISCUSSION TITLE
Froze on bootup start screen on Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by kshpdr** [utmapp/UTM#5711] [Linux]
**Describe the issue**  
I was using Ubuntu 22.04 but all of the sudden it stopped booting correctly and is just frozen on a bootup screen.

![Screenshot 2023-09-13 at 13 34 10](https://github.com/utmapp/UTM/assets/62385262/632ff251-9c12-4945-ab93-fbe9be5307fa)

**Configuration**  
* UTM Version: 4.3.5 (87)
* macOS Version: macOS Monterey 12.5 (21G72)
* Mac Chip (Intel, M1, ...): M1

**Crash log**  
No crash log, app didn't crash.

**Debug log**  
[debug.log](https://github.com/utmapp/UTM/files/12600387/debug.log)

**Comment by alibawada** [comment]
same here, I'm learning how to use linux, anyone can help?
<img width="674" alt="image" src="https://github.com/utmapp/UTM/assets/145784269/c1a14917-a359-468b-9a53-3b9c23f50835">

**Comment by wrmack** [comment]
I find Ubuntu 23.10 (Mantic Minotaur), even though beta, is much more stable using Apple virtualization.  I believe instability is usually to do with the kernel (though not necessarily in your case). Apple compatibility was introduced in kernel 6.2.  ARM64  iso is here: https://cdimage.ubuntu.com/daily-live/20230919/
